scripts/Tools/archive_metadata

```diff
@@ -1429,6 +1461,23 @@ def main_func(options):
 
     (case_dict, username, password) = initialize_main(options)
 
+    # check if query-expType argument is specified
+    if options.query_expType:
+        if case_dict['dryrun']:
+            logger.info('Dryrun - calling query_expdb_expType for supported experiment types')
+        else:
+            exp_types = query_expdb_expType(case_dict, username, password)
+            if exp_types:
+                for exp_type in exp_types:
+                    logger.info('expType: %s', exp_type["name"])
```

Comment:
This should probably just be print instead of logger.info

Reply:
Echoing what @jedwards4b said here: The general rule is: if the result of running the script is to produce some output for the user, then use `print`; if you're logging information on warnings/errors/general operating information, then use logger.

Reply:
fixed